INSTALL.md

```diff
@@ -65,7 +65,7 @@ curl -L -o cirrus https://github.com/cirruslabs/cirrus-cli/releases/latest/downl
 If you have [Go](https://golang.org/) 1.17 or newer installed, you can run:
 
 ```
-(cd && GO111MODULE=on go install github.com/cirruslabs/cirrus-cli/...@latest)
+cd && GO111MODULE=on go install github.com/cirruslabs/cirrus-cli/...@latest && cd -
```

Comment:
```suggestion
go install github.com/cirruslabs/cirrus-cli/...@latest
```

Let's cut it to just `go install github.com/cirruslabs/cirrus-cli/...@latest` since we're now using `go install` and [`GO111MODULE` is `on` by default](https://maelvls.dev/go111module-everywhere/#go111module-with-go-116) on newer Go versions.

Reply:
Great. I didn't know this.